app/script/entity/User.coffee

```diff
@@ -49,6 +49,7 @@ class z.entity.User
   ###
   constructor: (@id = '') ->
     @is_me = false
+    @is_bot = ko.observable false
```

Comment:
Does this need to be an observable? This would mean that a user can turn into a bot or vice versa!?